proto/api/v2/link_service.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package memos.api.v2;
+
+import "google/api/annotations.proto";
+
+option go_package = "gen/api/v2";
+
+service LinkService {
+  rpc GetLinkMetadata(GetLinkMetadataRequest) returns (GetLinkMetadataResponse) {
+    option (google.api.http) = {get: "/api/v2/metadata"};
+  }
+}
+
+// Metadata message
+message Metadata {
+  string title = 1;
+  string description = 2;
+  string image = 3;
+}
+
+// Request message for GetMetadata
+message GetLinkMetadataRequest {
+  string url = 1;
+}
+
+// Response message for GetMetadata
+message GetLinkMetadataResponse {
+  Metadata metadata = 1;
```

Comment:
```suggestion

message GetLinkMetadataRequest {
  string url = 1;
}

message GetLinkMetadataResponse {
  Metadata metadata = 1;
```

Reply:
@MehadND It's better to re-run `buf generate` as we modified protobuf codes.

Reply:
Ok will push the changes later as I'm away for few hours